core/includes/theme.maintenance.inc

```diff
@@ -15,6 +15,7 @@
  */
 function _backdrop_maintenance_theme() {
   global $theme, $theme_key;
+  $site_config = config('system.core');
```

Comment:
In this function it might make sense to move that config() down into the try{} - as that's the only place where this config is used.

There are two situations, where this config isn't needed at all, so it could be better to call it only if it's actually used.

Reply:
As always you are right, @indigoxela! Learning from you!